src/components/CardInfo.tsx

```diff
@@ -13,9 +15,10 @@ const CardInfo = ({ tailWide, number }: CardInfoProps) => {
     poster_path: '',
     original_title: '',
     title: '',
-    vote_average: '',
-    vote_count: '',
-    overview: ''
+    vote_average: 0,
+    vote_count: 0,
+    overview: '',
+    id: 0
```

Comment:
to my regret, it would be better to put null, but in this case we need to rewrite the typing for a large amount of code, since typing was prescribed everywhere only for a number :(

Reply:
this object is typeless as I see, this must be refactored, as providing 0 as id is not an option